packages/server/lib/server-base.ts

```diff
@@ -192,7 +192,7 @@ export abstract class ServerBase<TSocket extends SocketE2E | SocketCt> {
 
       // TODO: Can we just pass config.baseUrl regardless of project type?
       this._nodeProxy = httpProxy.createProxyServer({
-        target: projectType === 'ct' ? config.baseUrl : undefined,
+        target: config.baseUrl && projectType === 'ct' ? config.baseUrl : undefined,
```

Comment:
Unrelated to this change, saw the `todo` above... `Can we just pass config.baseUrl regardless of project type?`

Do we know the answer to this?

Reply:
I tried it out and it seems okay: https://github.com/cypress-io/cypress/pull/17200/commits/d06ecc009dcf4f531482f9d8ed3aadf3d9f6dd71

Reply:
Or not... it breaks some tests: https://app.circleci.com/pipelines/github/cypress-io/cypress/22009/workflows/6ef4a0b2-b286-438a-a56f-a8f6a88317a4/jobs/806471

I will put it back and see if that fixes them.

Reply:
Ok, looks like this conditional is required. I removed the comment.